### DISCUSSION TITLE
`pnpm.auditConfig.ignoreCves` and vulns without a CVE

### DISCUSSION THREAD

**Discussion by kamsar** [pnpm/pnpm#6204] [Q&A] [answered]
The https://pnpm.io/package_json#pnpmauditconfigignorecves specifically targets CVEs to ignore in `pnpm audit`.

Some vulnerabilities, such as https://github.com/advisories/GHSA-36jr-mh4h-2g58, show up in `pnpm audit` but have no assigned CVE. I've tried using the GHSA but that does not seem to match.

Does anyone know if that's intentional, or is this a bug?

**Comment by zkochan** [comment, marked as answer]
It is not a bug, it just wasn't implemented. Adding GHSA ID to that field probably doesn't make sense. So, we should probably create a new field or rename the existing one.

**Reply by pateljay1397** [reply]
@zkochan Do you have any alternative for this? if we want to add GHSA ids to avoid audit? Because some of the packages don't have CVEs.

**Comment by nezed** [comment]
Thanks for raising the topic, struggling with it right now for the latest version of Lerna
<img width="626" alt="image" src="https://github.com/pnpm/pnpm/assets/1659008/7da8e254-1eff-4650-b1e2-67490ed96f2a">
